### ISSUE TITLE
\new StaffGroup \with {} <<

### ISSUE THREAD

**Issue by mtarenskeen** [Philomelos/lilypond-musicxml2ly-dev#3]
I think this is a known issue: I see comments on this in the source of musicexp.py around line 2215.
I tried to convert a musicxml file and got a ly file with something like:

\new StaffGroup <<
    \with {systemStartDelimiter = #'SystemStartBrace } <<
        \new Staff <<

The "<<" after \with {...} shouldn't be there and isn't properly closed.
If I delete this my lilypond file compiles correctly.

**Comment by mtarenskeen** [comment]
Hi,

Any comment on this issue? The code around musicexp.py:2215 seems quite a mess to me. And it still is buggy. Needs some cleaning up? I have some pianomusic I need to convert, and musicxml2ly fails due this bug.

**Comment by Philomelos** [comment]
Yes, the code is suboptimal in both musicxml2ly and musicxml2ly-dev.  But you just have to do some small corrections  in the \score block of the generated .ly-file and you're good to go.  I won't have the time to improve this code in the forseeable future.  Patches are welcome!
